### PR TITLE
Add Angular Google Developer Experts on Twitter

### PR DESCRIPTION
**Pull request by johannes-weber**
closes issue #23

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +106,27 @@ Table of contents:
 * [`#ng-2 Slack Channel`](https://angularbuddies.slack.com/messages/ng-2/) on AngularBuddies ([Sign up](http://www.angularbuddies.com/))
 * [`#angular2 Slack Channel`](https://dartlang.slack.com/messages/angular2) on dartlang.slack.com ([Sign up](https://dartlang-slack.herokuapp.com/))
 
+### Angular Google Developer Experts
```
Comment (rolandjitsu) [inline README.md]: I think you should use `####` and there seems to already be a section of `Experts on Twitter` right under.

**Comment by jimthedev** [comment]
Can you add @jeffwelpley, too? https://twitter.com/jeffwhelpley/status/656088693212598272

**Comment by johannes-weber** [comment]
thx for the hint! - I took those who were listed yesterday on https://developers.google.com/experts/all/technology

**Comment by jeffwhelpley** [comment]
Thanks @jimthedev for the proactive shout out. They told me in an email that the info on the GDE site sometimes takes up to 2 weeks to get updated so no worries if you just wait until then.

**Comment by manekinekko** [comment]
@johannes-weber You're pointing to the main technology page. The title has been changed, we are now Web Technologies GDEs (which covers Angular of course) here: https://developers.google.com/experts/all/technology/web-technologies

**Pull request merged**